### PR TITLE
For #26450: Remove the star icon from Pocket sponsored stories

### LINKED ISSUE #26450: Remove the star icon from Pocket sponsored stories
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21199)

### PR DESCRIPTION
**Pull request by Mugurell**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made on Jira.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #26450

### CODE REVIEW

**Review by Amejia481 — APPROVED** [review]

File: app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt
Comment (gabrielluong) [inline app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:182]: While we are making this change, can you check the figma for this feature and see if this fits under any of the FenixTypography and switch to the appropriate text style if any? My guess would be Caption without verifying with the figma, but we should double check.

Comment (Mugurell) [reply app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:182]: The design from https://www.figma.com/file/wsvGxcKQEKgI4zPSzB0Kk6/SPOC-Placement-on-Android-%26-iOS?node-id=4%3A9020 doesn't quite match the available styles. Asked on Jira for maybe updating the design.

Comment (gabrielluong) [reply app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:182]: I think this is indeed using Caption when I inspected the node although it is using the iOS Caption given the SF Pro font. You can see Caption in the Typography section. I also compared the font size, weights, etc and I feel pretty confident we can unblock this change and move forward. Unfortunately, the Android designs are just a copy of the iOS components and the proper Android design tokens weren't used.

Screen Shot 2022-08-24 at 11 47 34 AM

Comment (Mugurell) [reply app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt:182]: Thank you.
That text seem to match `Caption` but the other two texts for the title and the sponsor name do not currently match any of our styles. Waiting for @Verdi 's feedback about them in https://mozilla-hub.atlassian.net/browse/FNXV2-21199?focusedCommentId=580916

**Comment by Mugurell** [comment]
`run-build - task :app:getDependencies FAILED` because of 
>  > Task :app:getDependencies
Failed to resolve OSS licenses for betaCompileClasspath.

Will unblock the landing queue and land this later.

**Comment by Mugurell** [comment]
Seems like merging is blocked at "Embarked in merge train"
with the following status

image


Restarting the CI.

**Comment by Mugurell** [comment]
Still enqueued with no progress. 
One more try.

**Pull request merged**
